src/contractor/contractor.cpp

```diff
@@ -412,8 +461,7 @@ std::size_t Contractor::LoadEdgeExpandedGraph(
         std::ofstream datasource_stream(datasource_indexes_filename, std::ios::binary);
         if (!datasource_stream)
         {
-            throw util::exception("Failed to open " + datasource_indexes_filename +
-                                  " for writing");
+            throw util::exception("Failed to open " + datasource_indexes_filename + " for writing");
         }
```

Comment:
just out of interest. Why are we writing in a function called `loadEdgeExpandedGraph`?